animate/adapt.py

```diff
@@ -61,14 +81,16 @@ def __init__(self, mesh, metric, name=None):
         if name is None:
             name = mesh.name
         self.name = name
+        self.comm = comm
```

Comment:
*** Marking all connected comments with these stars ***

Do the same as for `name`:
```python
if commis None:
    comm= mesh.comm
self.comm = comm
```

`mesh.comm` will be `COMM_SELF` because we loaded it by calling `CheckpointFile(..., comm=COMM_SELF)`

Reply:
Fixed in 83d8b81.